apps/aggregates/src/utils/index.ts

```diff
@@ -59,8 +73,12 @@ export function sortData(
  *
  * @param an array of objects
  */
-export function renameKey(data: Array, oldKey: String, newKey: String) {
-  return data.map((row) => {
+export function renameKey(
+  data: Record<string, any>,
+  oldKey: string,
+  newKey: string
+) {
+  return data.map((row: Record<string, any>) => {
```

Comment:
you can not do .map on objects, so either a bug or typing is incorrect

Reply:
Looks like a typing issue.